api/routes.go

```diff
@@ -37,4 +37,10 @@ func (s *server) routes() {
 	api.HandleFunc("/{account}/movers/{group}/{name}", s.MoverShowHandler).Methods(http.MethodGet)
 	api.HandleFunc("/{account}/movers/{group}/{name}", s.MoverDeleteHandler).Methods(http.MethodDelete)
 	// api.HandleFunc("/{account}/movers/{group}/{name}", s.MoverUpdateHandler).Methods(http.MethodPut)
+
+	api.HandleFunc("/{account}/movers/{group}/{name}/start", s.StartTaskHandler).Methods(http.MethodPost)
+	api.HandleFunc("/{account}/movers/{group}/{name}/stop", s.StopTaskHandler).Methods(http.MethodPost)
```

Comment:
PUT `/{account}/movers/{group}/{name}/runs/{id}`
```
{
   "State": "stop"
}
```

Reply:
As discussed, I am using current execution task ARN to stop. As only one task will be running at a time, we don't need of ID, please review. Thanks.